packages/server/postgres/queries/upsertTeamPromptResponses.ts

```diff
@@ -0,0 +1,25 @@
+import {DeepNonNullable} from '../../../client/types/generics'
+import getPg from '../getPg'
+import {
+  IUpsertTeamPromptResponsesQueryParams,
+  upsertTeamPromptResponsesQuery
+} from './generated/upsertTeamPromptResponsesQuery'
+
+export type InputTeamPromptResponses = DeepNonNullable<
+  IUpsertTeamPromptResponsesQueryParams['responses']
+>
+export type InputTeamPromptResponse = IUpsertTeamPromptResponsesQueryParams['responses'][0]
+
+export const upsertTeamPromptResponse = async (
+  response: InputTeamPromptResponse
+): Promise<number> => {
```

Comment:
+1 try to avoid declaring return types, sometimes it's unavoidable, but it forces good habits because the single source of truth is in the `return` statement